diff --git a/node_modules/unstated-persist/lib/unstated-persist.d.ts b/node_modules/unstated-persist/lib/unstated-persist.d.ts
index baa1c2a..5f6cedc 100644
--- a/node_modules/unstated-persist/lib/unstated-persist.d.ts
+++ b/node_modules/unstated-persist/lib/unstated-persist.d.ts
@@ -6,7 +6,7 @@ type PersistConfig = {
   storage: any, // @TODO add storage typing
 }
 
-export class PersistContainer<State extends object> extends Container{
+export class PersistContainer<State extends object> extends Container<State>{
   persist: PersistConfig
 }
 
